spec/system/ocr_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'OCR', :paperclip_processing, :sidekiq_inline do
+  include ProfileStories
+
+  subject { page }
+
+  let(:email)               { 'test@example.com' }
+  let(:password)            { 'password' }
+  let(:confirmed_at)        { Time.zone.now }
+  let(:finished_onboarding) { true }
+
+  before do
+    as_a_logged_in_user
+    visit root_path
+  end
+
+  it 'can recognize text in a media attachment' do
+    expect(subject).to have_css('div.app-holder')
+
+    within('.compose-form') do
+      attach_file('file-upload-input', Rails.root.join('spec', 'fixtures', 'files', 'text.png'), make_visible: true)
+
+      within('.compose-form__upload') do
+        click_on('Edit')
+      end
+    end
+
+    click_on('Detect text from picture')
```

Comment:
I assume the direct string usage here is because this string is in i18n ... but just the JS i18n, and we don't have a pattern of pulling those into these specs?

Reply:
Yeah